docker/osl.Dockerfile.dockerignore

```diff
@@ -0,0 +1,5 @@
+*
+!LICENSE
+!pom.xml
```

Comment:
not sure the pom.xml is needed the builder will probably override it (I'll test out of curiosity)

Reply:
Just tested it, we need to ignore.

Reply:
It needs to be ignore, I just tested it. To ensure it works we must use the pom.xml in the builder image